charts/gpu-device-plugin/Chart.yaml

```
@@ -2,5 +2,5 @@ apiVersion: v2
 name: intel-device-plugins-gpu
 description: A Helm chart for Intel GPU Device Plugin
 type: application
-version: 0.25.0
+version: 0.25.1
```

we don't have the versioning policy documented currently but for helm chart modifications prior to device plugins releases we use:
```suggestion
version: 0.25.1-helm0
```

---

Thanks for the insight. I was sure that in helm, for changes you use `version` and for managing image version `appVersion`